include/klee/Core/FunctionAnnotation.h

```diff
@@ -0,0 +1,46 @@
+#ifndef KLEE_FUNCTIONANNOTATION_H
+#define KLEE_FUNCTIONANNOTATION_H
+
+#include <vector>
+#include <map>
+#include <string>
+
+#include <nonstd/optional.hpp>
+#include <nlohmann/json.hpp>
+
+using nonstd::optional;
+using nonstd::nullopt;
+using json = nlohmann::json;
+
+namespace klee {
+
+struct FunctionAnnotation {
+  enum class StatementType {
+    Deref,
+    InitNull,
+  };
+
+  struct Statement {
```

Comment:
Maybe it will be better use separate classes instead of optional fields

Reply:
Done.